_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,639 @@
+---
+layout: docs
+title: Usage
+version: '1.0.0'
+categories:
+    - '1.0.0'
+---
+Schema
+======
+
+<!-- toc -->
+- [Intro](#intro)
+
+- [Data types](#data-types)
+
+  - [String](#string)
+
+  - [Boolean](#boolean)
+
+  - [Decimal](#decimal)
+
+  - [Long](#long)
+
+  - [Integer](#integer)
+
+  - [Short](#short)
+
+  - [Byte](#byte)
+
+  - [Double](#double)
+
+  - [Float](#float)
+
+  - [Timestamp](#struct)
+
+  - [Date](#timestamp)
+
+  - [Struct](#struct)
+
+  - [Array](#array)
+
+- [Metadata](#metadata)
+
+  - [sourcecolumn](#sourcecolumn)
+
+  - [default](#default)
+
+  - [pattern](#pattern)
+
+  - [timezone](#timezone)
+
+  - [decimal_separator](#decimal_separator)
+
+  - [grouping_separator](#grouping_separator)
+
+  - [minus_sign](#minus_sign)
+
+  - [allow_infinity](#allow_infinity)
+
+  - [radix](#radix)
+
+- [Parsing](#parsing)
+
+  - [Parsing timestamps and dates](#parsing-timestamps-and-dates)
+
+  - [Parsing numbers](#parsing-numbers)
+
+- [Defaults](#defaults)
+
+  - [Explicit default](#explicit-default)
+
+  - [Global default values](#global-default-values)
+
+  - [Local default values restrictions](#explicit-default-values-restrictions)
+<!-- tocstop -->
+
+Intro
+-----
```

Comment:
I am missing that "By defining schema u define which columns are omitted" or something of sorts.

Reply:
Will try to add something in this regard.